plat/kvm/efi.c

```diff
@@ -216,11 +216,26 @@ static int uk_efi_md_to_bi_mrd(struct uk_efi_mem_desc *const md,
 	return 0;
 }
 
-static void uk_efi_get_mmap(struct uk_efi_mem_desc **map, uk_efi_uintn_t *map_sz,
-			    uk_efi_uintn_t *desc_sz)
+static void uk_efi_get_mmap_and_exit_bs(struct uk_efi_mem_desc **map,
+					uk_efi_uintn_t *map_sz,
+					uk_efi_uintn_t *desc_sz)
 {
+	uk_efi_uintn_t map_key;
 	uk_efi_status_t status;
 	__u32 desc_ver;
+	__u8 retries = 0;
```

Comment:
Nitpick :laughing:. I like reverse xmas tree :christmas_tree: :santa:  variable declarations. So I would move this between `status` and `desc_ver`. :nerd_face: